foodkept/Pages/Register.cshtml.cs

```diff
@@ -122,10 +121,9 @@ public async Task<IActionResult> OnPostRestaurantAsync()
                 var assign_role = await userManager.AddToRoleAsync(user, "Restaurant");
 
 
-                if (result.Succeeded && assign_role.Succeeded) //&& emailResult.Result)
+                if (result.Succeeded && assign_role.Succeeded && emailResult.Result)
                 {
-                    await signInManager.SignInAsync(user, false);
```

Comment:
The restaurant will need to be confirmed by admin. Could you please explain how this confirmation will work on the admins end?

Reply:
Currently, restaurant's confirmation link is sent directly to our email "foodkepterino@gmail.com". At the moment, we are admins, so we confirm restaurant's registration by clicking on the link. Later, (I think) it would be nice verify every restaurant directly from our website.